snippets/permutations.md

```diff
@@ -0,0 +1,19 @@
+### permutations
```

Comment:
Isn't it the same as `anagrams` ?

Reply:
You're right ! God it's the first example and I did not seen it, sorry for the duplicate !